packages/app-project/src/shared/components/withRequireUser/withRequireUser.js

```diff
@@ -15,7 +15,7 @@ function storeMapper (stores) {
 
 function withRequireUser (WrappedComponent) {
   @inject(storeMapper)
-  class RequireUser extends React.Component {
+  class RequireUser extends Component {
```

Comment:
`React` is undefined here, in `withRequireUser`, but there's a bug in NextJS, with React 16, which defines `window.React` as a global variable.